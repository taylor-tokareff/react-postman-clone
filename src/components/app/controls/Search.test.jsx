import React from 'react';
import { render } from '@testing-library/react';
import SearchControls from './Search.jsx';

describe('SearchControls component', () => {
  it('renders searchControls', () => {
    const { asFragment } = render(
      <SearchControls
        url="plasscage.com"
        method="put"
        putBody="name: cage"
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
