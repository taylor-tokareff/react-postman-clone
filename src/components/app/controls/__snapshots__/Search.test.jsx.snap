// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchControls component renders searchControls 1`] = `
<DocumentFragment>
  <form
    class="actualForm"
  >
    <label
      for="query"
    />
    <input
      class="search"
      placeholder="URL"
      value="plasscage.com"
    />
    <div
      class="radioToolbar"
    >
      <input
        id="get"
        method="put"
        name="method"
        type="radio"
        value="GET"
      />
      <label
        for="get"
      >
        GET
      </label>
      <input
        id="post"
        method="put"
        name="method"
        type="radio"
        value="POST"
      />
      <label
        for="post"
      >
        POST
      </label>
      <input
        id="put"
        method="put"
        name="method"
        type="radio"
        value="PUT"
      />
      <label
        for="put"
      >
        PUT
      </label>
      <input
        id="patch"
        method="put"
        name="method"
        type="radio"
        value="PATCH"
      />
      <label
        for="patch"
      >
        PATCH
      </label>
      <input
        id="delete"
        method="put"
        name="method"
        type="radio"
        value="DELETE"
      />
      <label
        for="delete"
      >
        DELETE
      </label>
    </div>
    <div
      class="go"
    >
      <button
        class="goBut"
      >
        Go!
      </button>
    </div>
    <textarea
      class="raw"
      name="body"
      placeholder="Raw JSON Body"
    >
      name: cage
    </textarea>
  </form>
</DocumentFragment>
`;
